searchcore/src/vespa/searchcore/proton/server/lid_space_compaction_job_take2.cpp

```diff
@@ -44,16 +47,20 @@ CompactionJob::moveDocument(const search::DocumentMetaData & metaThen, std::shar
     if (metaNow.lid != metaThen.lid) return;
```

Comment:
__scanItr->getMetaData()_ is now called by the bucket executor thread. Is that really safe? We are looking into a structure that is traditionally updated by the docdb writer thread. @toregge I would like your opinion here as well.

Reply:
Yes, I came to the same conclusion myself in #16190